openapi/paths/key-value-stores/key-value-stores@{storeId}@records@{recordKey}.yaml

```diff
@@ -72,22 +71,15 @@ put:
     type defined by the `Content-Type` header and with encoding defined by the
     `Content-Encoding` header.
 
-
     To save bandwidth, storage, and speed up your upload, send the request
-    payload compressed with Gzip compression and
-
-    add the `Content-Encoding: gzip` header. It is possible to set up another
-    compression type with `Content-Encoding` request header.
+    payload compressed with Gzip compression and add the `Content-Encoding: gzip` 
+    header. It is possible to set up another compression type with `Content-Encoding` 
+    request header.
 
     Below is a list of supported `Content-Encoding` types.
 
-
     * Gzip compression: `Content-Encoding: gzip`
-
-
     * Deflate compression: `Content-Encoding: deflate`
-
-
     * Brotli compression: `Content-Encoding: br`
```

Comment:
Deleting the empty lines will cause the bullet list to collapse into a single line.

Reply:
IMHO we should fix this on the post-processing / rendering side.

This is valid Markdown, it should render properly.

cc @netmilk